noria-server/src/controller/migrate/mod.rs

```diff
@@ -209,35 +209,23 @@ impl<'a> Migration<'a> {
         if !self.readers.contains_key(&n) {
             let mut readers = Vec::with_capacity(num_replicas);
             for i in 0..num_replicas {
-                let replica_i = {
-                    if num_replicas == 0 {
-                        None
-                    } else {
-                        Some(i)
-                    }
-                };
-
                 // Make a reader node
-                let r = node::special::Reader::new(n, replica_i);
+                let r = node::special::Reader::new(n, i);
                 let r = if let Some(name) = name.clone() {
```

Comment:
Why does this need to `clone`, given that it's just passed to `format!` below?

Reply:
Ah, it did in the old code, since it used `name` directly, but I don't think it needs to any longer.

Reply:
👍